storage_backend_s3/README.rst

```diff
@@ -22,8 +22,6 @@ To install this module, you need to:
 Known issues / Roadmap
 ======================
 
-Update README with the last model of README when migration to v11 in OCA
-
 There is an issue with the latest version of `boto3` and `urllib3`
 - boto3 needs to be `boto3<=1.15.17` related with https://github.com/OCA/storage/issues/67
```

Comment:
Can you add this to the manifest then? on 15 you can force this :smile:

Reply:
Done!